scripts/pr_report.py

```
@@ -0,0 +1,276 @@
+PRIORITY = [
+    Category.FIX_CONFLICT,
+    Category.NEEDS_REBASE,
+    Category.NEEDS_REVIEW,
+    Category.READY_TO_MERGE,
+]
+
+
+# ── GitHub helpers ────────────────────────────────────────────────────────────
+
+def gh(*args):
+    result = subprocess.run(["gh"] + list(args), capture_output=True, text=True)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)